CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ The web, with simplicity.
 - [Anton Davydov] Don't configure mailers, if it's mailing is disabled for the project.
 - [Marcello Rocha] Ensure code reloading don't misconfigure mailer settings
 
+### Changed
```

Comment:
We only have three categories:

  - "Added" for new features
  - "Fixed" for fixed bugs
  - "Changed" for breaking changes